client/src/tests/mock-data/server.ts

```diff
@@ -0,0 +1,130 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { HttpCode } from 'aalto-grades-common/types';
+import { ResponseComposition, ResponseResolver, rest, RestContext, RestRequest } from 'msw';
+import { setupServer, SetupServer } from 'msw/node';
+
+import { mockAssessmentModels } from './mockAssessmentModels';
+import { mockAttainments } from './mockAttainments';
+import { mockCourses } from './mockCourses';
+import { mockFinalGrades } from './mockFinalGrades';
+import { mockInstances } from './mockInstancesWithStringDates';
+import { mockSisuInstances } from './mockSisuInstances';
+import { mockFormulas } from './mockFormulas';
+
+export function mockSuccess(data: unknown): ResponseResolver<RestRequest, RestContext> {
+  return async (_req: RestRequest, res: ResponseComposition, ctx: RestContext) => {
+    return res(
+      ctx.status(200),
+      ctx.json({
+        success: true,
```

Comment:
Do we even need the `success` attribute in our API?

Quickly checking through the code, only place we use the `success` value is during testing.
Also data and errors properties are never present same time in the backend response, so success is kind of redundant since we can determine the success of the request from properties. Axios doesn't use the `success` but the http code to determine when error must be thrown.

Some API specifications like [JSON api](https://jsonapi.org/format/#document-top-level) even states: "The members `data` and `errors` MUST NOT coexist in the same document." If we use this rule, then there is no need for `success` in my opinion.

Reply:
New issue: #381